### ISSUE TITLE
Is it really safe?

### ISSUE THREAD

**Issue by Miercoe** [Nexus42Dev/MacSploit#11]
Can you guys pls make an ah script that créate a the dylib file and the terminal executables so we can know what are inside of those files without compiling them JUST FOR SAFETY REASONS

**Comment by FireDasher** [comment]
Yes, it's really safe. It's the only Mac exploit since hydrogen screwed up their website.
